tests/Makefile

```diff
@@ -356,6 +356,9 @@ endif
 jenkins: xharness/xharness.exe
 	$(Q) $(SYSTEM_MONO) --debug $< $(XHARNESS_VERBOSITY) --jenkins --autoconf --rootdir $(CURDIR) --sdkroot $(XCODE_DEVELOPER_ROOT)
 
+jenkins-system: xharness/xharness.exe
```

Comment:
The name is not self-documentating and it's not on jenkins anymore right ?
minimally worth a comment above the target that describe it's use